common-lib/src/main/java/com/google/cloud/tools/intellij/feedback/PluginStatusBundle.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.intellij.feedback;
+
+import com.intellij.CommonBundle;
+
+import org.jetbrains.annotations.NonNls;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.PropertyKey;
+
+import java.lang.ref.Reference;
+import java.lang.ref.SoftReference;
+import java.util.ResourceBundle;
+
+/**
+ * Resource bundle for messages related to Google Feedback error reporting.
+ */
+public class PluginStatusBundle {
```

Comment:
I'm not sure about your first point regarding it being in the common-lib. Right now isn't it very tightly couple to checking if the old JB App Engine plugin is installed? This plugin is directly related to our cloud tools plugin and (in my opinion) has nothing to do with the account plugin.